pyaedt/filtersolutions_core/attributes.py

```diff
@@ -0,0 +1,1528 @@
+
+        Returns
+        -------
+        :enum:`DiplexerType`
+        """
+        type_string = self._dll_interface.get_string(self._dll.getDiplexerType)
+        return self._dll_interface.string_to_enum(DiplexerType, type_string)
+
+    @diplexer_type.setter
+    def diplexer_type(self, diplexer_type: DiplexerType):
+        string_value = self._dll_interface.enum_to_string(diplexer_type)
+        self._dll_interface.set_string(self._dll.setDiplexerType, string_value)
+
+    @property
+    def filter_multiple_bands_enabled(self) -> bool:
+        """Whether to enable the multiple bands table. The default is `False`.
+
+        Returns
+        -------
+        bool
+        """
+        filter_multiple_bands_enabled = c_bool()
+        status = self._dll.getMultipleBandsEnabled(byref(filter_multiple_bands_enabled))
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+        return bool(filter_multiple_bands_enabled.value)
+
+    @filter_multiple_bands_enabled.setter
+    def filter_multiple_bands_enabled(self, filter_multiple_bands_enabled: bool):
+        status = self._dll.setMultipleBandsEnabled(filter_multiple_bands_enabled)
+        pyaedt.filtersolutions_core._dll_interface().raise_error(status)
+
+    @property
+    def filter_multiple_bands_low_pass_frequency(self) -> str:
+        """The multiple bands low pass frequency of combined low pass and band pass filters. The default is `1GHz`.
+
+        Returns
+        -------
+        str
+        """
+        filter_multiple_bands_low_pass_freq_string = self._dll_interface.get_string(
+            self._dll.getMultipleBandsLowPassFrequency
+        )
+        return filter_multiple_bands_low_pass_freq_string
+
+    @filter_multiple_bands_low_pass_frequency.setter
+    def filter_multiple_bands_low_pass_frequency(self, filter_multiple_bands_low_pass_freq_string):
+        self._dll_interface.set_string(
+            self._dll.setMultipleBandsLowPassFrequency,
+            filter_multiple_bands_low_pass_freq_string,
+        )
+
+    @property
+    def filter_multiple_bands_high_pass_frequency(self) -> str:
+        """The multiple bands high pass frequency of combined high pass and band pass filters. The default is `1GHz`.
+
+        Returns
+        -------
+        str
+        """
+        filter_multiple_bands_high_pass_freq_string = self._dll_interface.get_string(
+            self._dll.getMultipleBandsHighPassFrequency
+        )
+        return filter_multiple_bands_high_pass_freq_string
+
+    @filter_multiple_bands_high_pass_frequency.setter
+    def filter_multiple_bands_high_pass_frequency(self, filter_multiple_bands_high_pass_freq_string):
+        self._dll_interface.set_string(
+            self._dll.setMultipleBandsHighPassFrequency,
+            filter_multiple_bands_high_pass_freq_string,
+        )
+
+    @property
+    def order(self) -> int:
+        """Order of filter. The default is `5`.
```

Comment:
```suggestion
        """Order of the filter. The default is ``5``.
```
Order for what? Order in which the filter is applied?